src/js/components/messagehandler/messagehandler.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Snackbar from '../snackbar/snackbar';
+
+class MessageHandler extends Component {
```

Comment:
This component doesn't handle any state, it could just be a [functional component](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc).

Reply:
Done.